### PR TITLE
Compute client library update for encryption settings, forceUpdateTag and VMInstanceIDs.

### PR DESCRIPTION
**Pull request by hyonholee**
Adding encryption settings, forceUpdateTag and VMInstanceIDs properties to Compute client library.

https://github.com/Azure/azure-rest-api-specs/pull/1422

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs [hunk 1]
```diff
@@ -598,9 +614,14 @@ public static OperationStatusResponse UpdateInstances(this IVirtualMachineScaleS
             /// <param name='vmScaleSetName'>
             /// The name of the VM scale set.
             /// </param>
-            public static OperationStatusResponse Reimage(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName)
+            /// <param name='instanceIds'>
+            /// The virtual machine scale set instance ids. Omitting the virtual machine
+            /// scale set instance ids will result in the operation being performed on all
+            /// virtual machines in the virtual machine scale set.
+            /// </param>
+            public static OperationStatusResponse Reimage(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, IList<string> instanceIds = default(IList<string>))
```
Comment (shahabhijeet) [inline src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: @hyonholee breaking change, will need to bump up major version.

File: src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs [hunk 2]
```diff
@@ -616,12 +637,17 @@ public static OperationStatusResponse Reimage(this IVirtualMachineScaleSetsOpera
...
+            /// <param name='instanceIds'>
+            /// The virtual machine scale set instance ids. Omitting the virtual machine
+            /// scale set instance ids will result in the operation being performed on all
+            /// virtual machines in the virtual machine scale set.
+            /// </param>
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<OperationStatusResponse> ReimageAsync(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<OperationStatusResponse> ReimageAsync(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, IList<string> instanceIds = default(IList<string>), CancellationToken cancellationToken = default(CancellationToken))
```
Comment (shahabhijeet) [inline src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: @hyonholee breaking change

File: src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs [hunk 3]
```diff
@@ -616,12 +637,17 @@ public static OperationStatusResponse Reimage(this IVirtualMachineScaleSetsOpera
...
             {
-                using (var _result = await operations.ReimageWithHttpMessagesAsync(resourceGroupName, vmScaleSetName, null, cancellationToken).ConfigureAwait(false))
+                using (var _result = await operations.ReimageWithHttpMessagesAsync(resourceGroupName, vmScaleSetName, instanceIds, null, cancellationToken).ConfigureAwait(false))
```
Comment (shahabhijeet) [inline src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: @hyonholee breaking chagne

File: src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs [hunk 4]
```diff
@@ -640,9 +666,14 @@ public static OperationStatusResponse Reimage(this IVirtualMachineScaleSetsOpera
             /// <param name='vmScaleSetName'>
             /// The name of the VM scale set.
             /// </param>
-            public static OperationStatusResponse ReimageAll(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName)
+            /// <param name='instanceIds'>
+            /// The virtual machine scale set instance ids. Omitting the virtual machine
+            /// scale set instance ids will result in the operation being performed on all
+            /// virtual machines in the virtual machine scale set.
+            /// </param>
+            public static OperationStatusResponse ReimageAll(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, IList<string> instanceIds = default(IList<string>))
```
Comment (shahabhijeet) [inline src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: @hyonholee breaking change

File: src/SDKs/Compute/Management.Compute/Generated/Models/Page1.cs [hunk 5]
```diff
@@ -2,17 +2,17 @@
 // Licensed under the MIT License. See License.txt in the project root for
 // license information.
 //
-// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Code generated by Microsoft (R) AutoRest Code Generator.
```
Comment (shahabhijeet) [inline src/SDKs/Compute/Management.Compute/Generated/Models/Page1.cs]: @hyonholee can you explain why you have two similar model types Page and Page1?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
I don't see any test coverage for this PR.
Also can you describe what this PR is all about.
Giving some idea as to what is the change, helps in understanding why this PR was open.

File: src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs (hunk 1 above)
Comment (hyonholee) [inline src/SDKs/Compute/Management.Compute/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: @shahabhjieet This new parameter is optional.  Why is this a breaking change?

File: src/SDKs/Compute/Management.Compute/Generated/Models/Page1.cs (hunk 5 above)
Comment (hyonholee) [inline src/SDKs/Compute/Management.Compute/Generated/Models/Page1.cs]: @shahabhijeet  The file is generated by autorest.  I don't know why autorest creates these files.

**Pull request merged**